server/perfkit/explorer/handlers/templates/dashboard-admin.html

```diff
@@ -2,20 +2,35 @@
 <html lang="en" ng-app="explorer" ng-controller="DashboardAdminPageCtrl as pageCtrl">
 <head>
   <title>Perfkit - Dashboard Administration</title>
+
+  <base href="/">
+
   <!-- JQuery & Angular -->
-  <script src="https://ajax.googleapis.com/ajax/libs/jquery/2.1.1/jquery.min.js"></script>
-  <script src="https://ajax.googleapis.com/ajax/libs/angularjs/1.2.18/angular.min.js"></script>
-  <script src="https://ajax.googleapis.com/ajax/libs/angularjs/1.2.18/angular-mocks.js"></script>
+  <script src="https://ajax.googleapis.com/ajax/libs/jquery/2.1.4/jquery.min.js"></script>
+  <script src="https://ajax.googleapis.com/ajax/libs/angularjs/1.3.15/angular.min.js"></script>
+  <script src="https://ajax.googleapis.com/ajax/libs/angularjs/1.3.15/angular-animate.min.js"></script>
+  <script src="https://ajax.googleapis.com/ajax/libs/angularjs/1.3.15/angular-aria.min.js"></script>
+  <script src="https://ajax.googleapis.com/ajax/libs/angularjs/1.3.15/angular-mocks.js"></script>
```

Comment:
Although broken atm, you can use mock=true in the url, and be served mock dashboards, widgets and data instead of live ones.  This was used for early testing, though I've let it fall by the wayside.